### ISSUE TITLE
[NBug] емкость меньше текущего размера.Имя параметра: value

### ISSUE THREAD

**Issue by LiDReSaR** [gitextensions/gitextensions#8190]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.ArgumentOutOfRangeException: емкость меньше текущего размера.
Имя параметра: value
   в System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   в System.Collections.Generic.List`1.set_Capacity(Int32 value)
   в GitUI.FileStatusList.SetDiffs(IReadOnlyList`1 revisions, Func`2 getRevision)
   в GitUI.UserControls.CommitDiff.SetRevision(ObjectId objectId, String fileToSelect)
   в GitUI.HelperDialogs.FormCommitDiff..ctor(GitUICommands commands, ObjectId objectId)
   в GitUI.RevisionGridControl.<>c__DisplayClass169_0.<ViewSelectedRevisions>g__ProvideForm|1()
   в GitUI.GitUICommands.ShowModelessForm(IWin32Window owner, Boolean requiresValidWorkingDir, EventHandler`1 preEvent, EventHandler`1 postEvent, Func`1 provideForm)
   в GitUI.RevisionGridControl.ViewSelectedRevisions()
   в GitUI.RevisionGridControl.OnGridViewDoubleClick(Object sender, MouseEventArgs e)
   в System.Windows.Forms.Control.OnMouseDoubleClick(MouseEventArgs e)
   в System.Windows.Forms.DataGridView.OnMouseDoubleClick(MouseEventArgs e)
   в System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в System.Windows.Forms.DataGridView.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Additional information
(Disabled showing of parent diffs)
After DblClick on Merge Commit

## Environment
- Git Extensions 3.4.0.9381
- Build 096b4d56709e15a67b42a471bbe81ce3c95ae7f5
- Git 2.27.0.windows.1
- Microsoft Windows NT 6.2.9200.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

**Comment by LiDReSaR** [comment]
1) Open GitExtensions Settings -> Detailed -> Diff Viewer.
2) Disable checkbox "Show file differences for all parents in browser dialog" and press Apply/OK.
3) Double Click on some Merge Commit.

**Comment by RussKie** [comment]
@gerhardol @mstv

**Comment by mstv** [comment]
Duplicate of #8097. Resolved by #8101 in the [pre-release 3.4](https://github.com/gitextensions/gitextensions/releases/tag/v3.4).

**Comment by RussKie** [comment]
👍
